### DISCUSSION TITLE
Allowed keycodes for Lua plugin

### DISCUSSION THREAD

**Discussion by dinojr** [xournalpp/xournalpp#4368] [Q&A]
Is there a reference for the keys available for use as accelerators in a lua plugin ?

+ I'd like to use the "End" key but the <End> code doesn't seem to work
+  \<Ctrl\>! doesn't work either

**Comment by rolandlo** [comment]
There is mostly [this page](https://docs.gtk.org/gtk3/func.accelerator_parse.html) and the [code](https://github.com/GNOME/gtk/blob/main/gtk/gtkaccelgroup.c).
